### ISSUE TITLE
Unable to open /dev/ttyUSB1 after upgrade to v3.0.1

### ISSUE THREAD

**Issue by TRIROG** [pyserial/pyserial#67]
Upgraded pyserial to v 3.0.1
list available COM ports:
```
$ python -m serial.tools.list_ports
/dev/ttyAMA0
/dev/ttyUSB0
/dev/ttyUSB1
3 ports found
```

running 
```$ python -m serial.tools.miniterm /dev/ttyUSB0```
opens miniterm:
```
--- Miniterm on /dev/ttyUSB0  9600,8,N,1 ---
--- Quit: Ctrl+] | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H ---
at
OK
```
works fine.

However running 
```$ python -m serial.tools.miniterm /dev/ttyUSB1```
returns 
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/lib/python2.7/dist-packages/serial/tools/miniterm.py", line 878, in <module>
    main()
  File "/usr/local/lib/python2.7/dist-packages/serial/tools/miniterm.py", line 834, in main
    serial_instance.open()
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 311, in open
    self._update_dtr_state()
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 605, in _update_dtr_state
    fcntl.ioctl(self.fd, TIOCMBIS, TIOCM_DTR_str)
IOError: [Errno 32] Broken pipe
```
After downgrade to pyserial 2.7 everything works fine:
```
$  python -m serial.tools.miniterm /dev/ttyUSB1
--- Miniterm on /dev/ttyUSB1: 9600,8,N,1 ---
--- Quit: Ctrl+]  |  Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H ---
at
OK

^BOOT:31506079,0,0,0,77

^RSSI:18

^RSSI:18

^BOOT:31506079,0,0,0,77
```

In my case The ttyUSB1 is the status serial port of a USB 3g modem Huawei K3565, which the modem uses to report its states and other events (signal strengts, received calls etc)

**Comment by zsquareplusc** [comment]
This seems to be related to #59, but it is reporting a different error. Might still be the driver complaining about the use of control signals when it does not support them (?).
Anyway the fix for one should also work for the other.

**Comment by davidjb** [comment]
I had the same Broken Pipe issue with a ZTE K3571 and upgrading to the latest `master` version of pyserial (from v3.0.1) resolves it

**Comment by TRIROG** [comment]
solved by passing `dsrdtr=True` and `rtscts=True` to serial.Serial() ... as described [here](https://github.com/bewest/decoding-carelink/pull/171)

**Comment by zsquareplusc** [comment]
released 3.1 where an error setting control lines in open() is ignored (but not in later calls).
